code-of-conduct_events

```diff
@@ -0,0 +1,104 @@
+_This is the addendum to our [Code of Conduct](code-of-conduct.md) for in-person events and gatherings._
+## Events
+
+In addition to the guidance above, please see the following process for IPFS-sponsored in-person events.
+
+### Conflict Resolution
+
+1. Initial Incident
+
+    If you are being harassed, notice that someone else is being harassed,
+    or have any other concerns, __and you feel comfortable speaking with
+    the offender__, please inform the offender that they have affected you 
+    negatively. Oftentimes, the offending behavior is unintentional, and the 
+    accidental offender and offended will resolve the incident by having 
+    that initial discussion.
+    
+    IPFS recognizes that there are many reasons speaking directly to
+    the offender may not be workable for you (including but not limited to
+    unfamiliarity with the conference or its participants, [lack of spoons](http://www.butyoudontlooksick.com/wpress/articles/written-by-christine/the-spoon-theory/),
+    and concerns for personal safety). If you don't feel comfortable 
+    speaking directly with the offender for any reason, skip straight to 
+    step 2.
+
+2. Escalation 
+
+    If the offender insists that they did not offend, if offender is
+    actively harassing you, or if direct engagement is not a good option
+    for you at this time, then you will need a third party to step in.
+
+    If you are at a conference or other event, find the event organizer or
+    staff person. How to identify an event organizer or staff person will be listed 
+    clearly on individual event repos/webpages. Or, you can always email abuse@ipfs.io.
+
+3. Wider community response to Incident:
+
+    If incident results in corrective action, the community should support
+    the decision made by the Help in Step 2 if they choose corrective action,
+    like ending a talk early or banning from the event and other IPFS 
+    communication channels, as well as support those harmed by the incident, 
+    either publicly or privately (whatever individuals are comfortable with).
+
+    If the Help in Step 2 run into issues implementing the CoC, then the
+    Help should come to the community with these issues and the community
+    should revise the CoC as they see fit.
+
+    In Real Life people will have opinions about how the CoC is enforced.
+    People will argue that a particular decision was unfair, and others will
+    say that it didn't go far enough. We can't stop people having
+    opinions, but what we could do here is have constructive discussions
+    that lead to something tangible (affirmation of decision, change in CoC,
+    modify decision, etc).
```

Comment:
This whole section is a bit weird. On one side it is telling the community that they "should support the decision" and the same time saying that "we should discuss it because we can't force you agree". But it doesn't set a process about how to do that, or when, or where.

I'd fully remove this. Seems obvious, it's poorly written and adds little value to the Code.